samples/snippets/set_options_test.py

```diff
@@ -39,3 +39,6 @@ def test_bigquery_dataframes_set_options() -> None:
     # [END bigquery_dataframes_set_options]
     assert bpd.options.bigquery.project == PROJECT_ID
     assert bpd.options.bigquery.location == REGION
+
+    bpd.close_session()
+    bpd.options.reset()
```

Comment:
Let's put anything that changes global options in a try/finally block so that this cleanup can run no matter what.

Reply:
SG!